claripy/simplifications.py

```diff
@@ -280,6 +298,13 @@ def ne_simplifier(a, b):
             # elif b._claripy.is_true(b.args[1] == a) and b._claripy.is_true(b.args[2] == a):
             #	  return b._claripy.false
 
+        # 1 ^ expr != 0     ->   expr == 0
+        if a.op == '__xor__' and b.args[1] == 1 and b.args[0] == 0:
+            if a.args[1].op == "BVV" and a.args[1].args[0] == 1 and a.args[0].size() == 1:
+                return a.args[0] == 0
```

Comment:
The above 3 lines read to me as transform:
`( x ^ BVV[1, ...] ^ ... ) == b.op[0, 1, ...]` into `x == 0` where `x.size() = 1`. This seems wrong.

Reply:
The following two lines read similarly

Reply:
I stopped reviewing here.

Reply:
The comment here does not match what this code is doing.

Reply:
Is it meant to be `expr ^ 1 == b -> expr == b` ? If this is the case, I'm not sure why `b.op` matters nor why `a.args[0]` matters.

Reply:
I again stopped reviewing here.